pylab/website/tests/test_comments.py

```diff
@@ -0,0 +1,41 @@
+import datetime
+
+from django_webtest import WebTest
+
+from django.contrib.auth.models import User
+
+from django_comments.models import Comment
+
+from pylab.core.models import Project
+
+
+class CommentsTests(WebTest):
+
+    def setUp(self):
+        self.user = User.objects.create_user('u1')
+        self.project = Project.objects.create(
+            author=self.user,
+            title='Test project',
+            description='Description',
+            created='2015-08-13'
+        )
+
+    def test_add_comment(self):
+        resp = self.app.get('/projects/test-project/', user=self.user)
+        now = datetime.datetime.now()
+        comment = Comment.objects.create(
```

Comment:
Comment should be created using comment form provided in the ```/projects/test-project/``` page. You should fill form fields and commit that form instead of creating comment directly in database. If you fill and save form, than you test if form, urls, views and configuration are working correctly. Currently this test only checks if page opens without errors when comment exist and when it doesn't.